extensions/blocks/podcast-player/edit.js

```diff
@@ -203,20 +219,55 @@ const PodcastPlayerEdit = ( {
 						onChange={ value => setAttributes( { showEpisodeDescription: value } ) }
 					/>
 				</PanelBody>
+				<PanelColorSettings
+					title={ __( 'Color Settings', 'jetpack' ) }
+					colorSettings={ [
+						{
+							value: primaryColorProp.color,
+							onChange: setPrimaryColor,
+							label: __( 'Primary Color', 'jetpack' ),
+						},
+						{
+							value: secondaryColorProp.color,
+							onChange: setSecondaryColor,
+							label: __( 'Secondary Color', 'jetpack' ),
+						},
+						{
+							value: backgroundColorProp.color,
+							onChange: setBackgroundColor,
+							label: __( 'Background Color', 'jetpack' ),
+						},
+					] }
+				>
+					<ContrastChecker
```

Comment:
I wasn't able to trigger the contrast checker on my testing site. What's the best way to test this? I have tried setting all three colors to the same custom color (`#fff`) and it still didn't show up for me.

Reply:
let me take a look. Also, I think we'll have to iterate on it.